docs/en/engines/table-engines/special/distributed.md

```diff
@@ -50,6 +50,8 @@ Clusters are set like this:
             <!-- Optional. Whether to write data to just one of the replicas. Default: false (write data to all replicas). -->
             <internal_replication>false</internal_replication>
             <replica>
+                <!-- Optional. Priority of the replica for load balancing (see also load_balancing setting). Default: 1 (less value has more priority). -->
+                <priority>1</priority>
```

Comment:
Always that silly problem with what has bigger priority 0 or 255.

If you prefer to have less value = bigger priority, please ensure that we support here negative numbers. (user can want to increase the priority "above" 1 or zero).

Reply:
>ensure that we support here negative numbers

There is simple sort, should work